change/@uifabric-azure-themes-2021-03-01-09-34-59-dark.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Default Button Dark Theme Edits",
+  "packageName": "@uifabric/azure-themes",
+  "email": "aidanmc95@gmail.com",
+  "dependentChangeType": "none",
```

This needs to be type `patch`; otherwise the change won't get published. (See [this page](https://github.com/microsoft/fluentui/wiki/Change-Files) for info on choosing the correct change type.)
```suggestion
  "type": "patch",
  "comment": "Default Button Dark Theme Edits",
  "packageName": "@uifabric/azure-themes",
  "email": "aidanmc95@gmail.com",
  "dependentChangeType": "patch",
```